src/grt/src/fastroute/src/utility.cpp

```diff
@@ -2302,6 +2304,107 @@ int FastRouteCore::edgeShiftNew(Tree& t, int net)
   return (numShift);
 }
 
+odb::Rect FastRouteCore::globalRoutingToBox(const GSegment& route)
+{
+  int init_x, init_y;
+  int final_x, final_y;
+
+  if (route.init_x < route.final_x) {
+    init_x = route.init_x;
+    final_x = route.final_x;
+  } else {
+    init_x = route.final_x;
+    final_x = route.init_x;
+  }
+
+  if (route.init_y < route.final_y) {
+    init_y = route.init_y;
+    final_y = route.final_y;
+  } else {
+    init_y = route.final_y;
+    final_y = route.init_y;
+  }
```

Comment:
Simplify to:
const auto [init_x, final_x] = std::minmax(route.init_x, route.final_x);
const auto [init_y, final_y] = std::minmax(route.init_y, route.final_y);

Reply:
The change was applied